translatable.yml

```diff
@@ -0,0 +1,14 @@
+type: RoR
+
+dictionary:
+ type: google_sheet
+ sheet_id: 1B9RnT3vscNM8YGtIpzqNlSPCQUdHrcJ3cGPSGOtHmoA
```

Comment:
Is it ok to have this in a public repo?

Reply:
they shouldn't be able to edit it, it's only for download